README.md

```diff
@@ -1,46 +1,49 @@
 # iris-project
 
-## How to update the site
+## How do I update the site?
 
 Updating is just a matter of commiting and pushing your changes. Github-pages will automatically observe the change and in a matter of seconds the main home-page will be updated. 
 
-## How to add publications
+## How do I test the site locally?
 
-To add a new publication find
-```html
-    <dl class="ref">
+Assuming you have a working and decently recent Ruby installation with `bundler` (`gem install bundler`) you should be able to set everything up by running at the root of the repository to install all dependencies (including Jekyll).
```

Comment:
Jekyll is shipped with Debian; so it could be worth mentioning that often it is easier to just install that and then do `jekyll serve`?

Reply:
Sure, feel free to add whatever information you think is relevant. However, I think you need specific gems for GitHub pages, so I'm not 100% sure what is the procedure in that case. Using `bundler` seems to be a reliable way to get the specific deps you need, and to avoid mess up the rest of your system since everything can be installed locally at the root of the repo.

Reply:
Yeah, looks like the one shipped with Debian does not work. There is some sanity check somewhere comparing against the gemfile.

Reply:
OK let's stick with the local install of deps then.